### ISSUE TITLE
Regarding fine-tuning the model

### ISSUE THREAD

**Issue by sayan-dey** [asw0316/binshot#1]
Hi,  
I was trying to finetune the model `~/binshot/models/pretrain_full/model_bert/bert_ep19.model` using a P100 GPU. It is taking about 12 hours per epoch. Is this expected? Can you please let me know the expected time and number of epochs it takes for convergence? If I am not wrong, the final model `models/downstream/pred.test.all_all`, which is used for generating the result, is not there in the github repo. It would be very much helpful if it can be shared. 
Any inputs would be appreciated.

**Comment by SeonggwanAhn** [comment]
(1) pretraining: We didn't take that much time to pretrain. If you didn't change any hyper-parameters in ```hparams.py```, I wish you to check the input data. And we used the model trained on 20th epoch, which is ```bert_ep19.model```

(2) ```pred.test.all_all``` is not a final model. It is the result file of inference on the test set. By executing [this](https://github.com/asw0316/binshot#finetuning--evaluation), You can see the file.

Thanks

**Comment by asw0316** [comment]
To add to @SeonggwanAhn 's answer, I remember that the entire process took about 2-3 days with the experimental environment described in the paper.

I recommend you to check if your binaries are too big, or your GPU is running properly.

**Comment by sayan-dey** [comment]
Hi. Thanks for the help. I was able to successfully pretrain as well as fintune the model.
Now I was trying to generate the corpus for some other set of binaries.

For that, `corpusgen.py` is the code if I understood correctly. But I am facing few issues while using it.

1) It needs a pickle directory for the corpus as argument but I am unable to understand what this should be.
2) Also, it has reference to `elf.py` which is trying to `import conf as C` but I can't find any file called `conf.py`.

It will be really great if you can help in this regard. 

Thanks.

**Comment by asw0316** [comment]
Hello again,

1. Do not skip the preceding steps. README says that you should execute _gen_ida.sh_ and _gen_norm.sh_ before executing _corpusgen.py_.
2. Install the library with pip.
This library should have been installed in the prerequisite installation.
(pip install -r requirements.txt)
Check if the prerequisites are installed properly.

**Comment by cxxz16** [comment]
i have install the conf==0.4.1 and can import from python. but when i read the docs of conf, i don't think these two conf package are same. I think there is a conf.py file you wrote yourself that was not uploaded. I would greatly appreciate it if you could solve this problem
![image](https://github.com/asw0316/binshot/assets/47975519/46356b4f-6f18-434f-a60d-e4ef5bd698c7)

update: i have provide a method that maybe true in another issue.
